### PR TITLE
Add new frame generator example

### PR DESCRIPTION
**Pull request by oscargm98**

### CODE REVIEW

File: examples/example_frame_generator.c [hunk 1]
```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2018 Francesc Alted, Aleix Alcacer.
```
Comment (FrancescAlted) [inline examples/example_frame_generator.c]: No need to add the copyright of the individuals above. We are the Blosc Development Team now :-)

File: examples/example_frame_generator.c [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    err = cyclic();
+    if (err != CATERVA_SUCCEED) {
+        printf("\n Cyclic error: %d", err);
+    }
+    err = many_matches();
+    if (err != CATERVA_SUCCEED) {
+        printf("\n Many_matches error: %d", err);
+    }
+    return err;
+}
```
Comment (FrancescAlted) [inline examples/example_frame_generator.c]: A newline is desirable at the end of the file.

**Review by FrancescAlted — CHANGES_REQUESTED** [review]

File: examples/example_frame_generator.c (hunk 1 above)
Comment (oscargm98) [reply examples/example_frame_generator.c]: I copied it from the other examples. Should I delete the sentence from all the Caterva examples?

File: examples/example_frame_generator.c (hunk 2 above)
Comment (oscargm98) [reply examples/example_frame_generator.c]: solved!

File: examples/example_frame_generator.c (hunk 1 above)
Comment (FrancescAlted) [reply examples/example_frame_generator.c]: Nah. It is just that copyright lines in new files only should care about The Blosc Development Team.

Comment (oscargm98) [reply examples/example_frame_generator.c]: Ok, it is done for this example.

**Pull request merged**